app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,30 @@
+$Event-color-background: Theme-color(background, default)
+$Event-color-borderTop: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
+$Event-height: 150px
+$Event-spacing-base: $Theme-spacing-base
+$Event-text-title-color: Theme-color(text, default)
+$Event-width: 280px
+
+.Event
+  background-color: $Event-color-background
+  border-radius: 3px
+  border-top: 5px solid $Event-color-borderTop
+  box-shadow: 0 2px 10px -3px gray
+  display: inline-block
+  height: $Event-height
+  margin: $Event-spacing-base
```

Comment:
I can't came up with a solution for this to work. I know that is the ancestor that is responsible to the positioning, but can't work it out, because for this case I have two kinds of arrangement. (aligned to right on events list and on center for the button to create a new button.)